go/libkb/secret_store.go

```diff
@@ -129,6 +129,39 @@ func GetConfiguredAccounts(m MetaContext, s SecretStoreAll) ([]keybase1.Configur
 			IsCurrent: username.Eq(currentUsername),
 		}
 	}
+
+	// Get the full names
+
```

Comment:
nit: I actively fight against empty lines after comments / func decls, while this function is written in this style I _really_ don't like it. It's OK though.

Reply:
AIUI, comments without a line break following them ought to be for the following line, whereas comments with a line break ought to be for the following section. What would you write instead?

Reply:
I suppose this makes sense, I've _never_ seen it in the wild though!